packages/alfa-rules/src/sia-r69/diagnostic/contrast.ts

```diff
@@ -0,0 +1,125 @@
+import { Diagnostic } from "@siteimprove/alfa-act";
```

Comment:
This might need to be moved to a common folder once we get a rule for "color contrast (enhanced)".

Reply:
Yep, I imagine shared diagnostics moving into `common/diagnostic`.